src/Mod/Draft/draftutils/gui_utils.py

```diff
@@ -64,6 +64,8 @@ def get_3d_view():
         Return `None` if the graphical interface is not available.
     """
     if App.GuiUp:
+        import FreeCADGui as Gui
+        from pivy import coin
```

Comment:
@Syres916 those modules are already imported in this lines
https://github.com/FreeCAD/FreeCAD/blob/a049b9118be81246b754c8cd963e434ce8a6576b/src/Mod/Draft/draftutils/gui_utils.py#L48-L51